### PR TITLE
Updated readme.md for HeapSort

### PR DESCRIPTION
**Pull request by nitishtw**
Thank you for your contribution. Please provide the details requested below.

## ISSUE NUMBER

Please provide a link to the issue this pull request addresses.

## SHORT DESCRIPTION

Please provide a short description of the changes.

## TESTING

Please provide the necessary steps to test the solution.

### CODE REVIEW

File: Competitive Coding/Sorting/Heap Sort/Readme.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-{3) The heap is reconstructed after each removal.
 
-Time Complexity: Time complexity of heapify is O(logn). 
-Time complexity of BUILD-MAX-HEAP(A) is O(n) 
-and we run max_heapify N−1 times in heap_sort function,
-therefore complexity of heap_sort function in each case (Best, worst and average) is O(nlogn)
+(2) Then a sorted array is created by repeatedly removing the largest/smallest element from the heap, and inserting it into the array.
+
+(3) The heap is reconstructed after each removal.
+
```
Comment (prateekiiest) [inline Competitive Coding/Sorting/Heap Sort/Readme.md]: use `*` instead of (1)/ (2)

File: Competitive Coding/Sorting/Heap Sort/Readme.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
-# Heap sort is a comparison based sorting technique based on Binary Heap data structure.
-# Heapsort is an in-place algorithm, but it is not a stable sort.
-# Heap Sort is one of the best sorting methods being in-place and with no quadratic worst-case scenarios.
-# In max-heaps, maximum element will always be at the root. Heap Sort uses this property of heap to sort the array.
+Heap sort is a comparison based sorting technique based on Binary Heap data structure. Heapsort is an in-place algorithm, but it is not a stable sort. Heap Sort is one of the best sorting methods being in-place and with no quadratic worst-case scenarios. In max-heaps, maximum element will always be at the root. Heap Sort uses this property of heap to sort the array.
```
Comment (prateekiiest) [inline Competitive Coding/Sorting/Heap Sort/Readme.md]: create a heading, 
### Documentation for HeapSort 

using ###

**Review by prateekiiest — CHANGES_REQUESTED** [review]
Post a gif like other sorting algos

**Comment by AdvikEshan** [comment]
Sir, what should be the heading...and should i embed the gif into readme

**Comment by prateekiiest** [comment]
yes , I already said the heading there in the bold letters.

Also upload the gif and post a link such that its visible in markdown preview

**Review by nitishtw — COMMENTED** [review]
readme updated

File: Competitive Coding/Sorting/Heap Sort/Readme.md (hunk 2 above)
Comment (nitishtw) [reply Competitive Coding/Sorting/Heap Sort/Readme.md]: done

File: Competitive Coding/Sorting/Heap Sort/Readme.md (hunk 1 above)
Comment (nitishtw) [reply Competitive Coding/Sorting/Heap Sort/Readme.md]: done!

**Comment by nitishtw** [comment]
Sir, i have made heading in bold and uploaded the gif, posted the link of
image in readme too...!!

**Review by prateekiiest — APPROVED** [review]

**Comment by prateekiiest** [comment]
Looks good 🎆 

Thanks for the contribution :1st_place_medal:

**Comment by nitishtw** [comment]
Thank you so much sir!! 😄😄

**Pull request merged**